src/scripts/main.js

```diff
@@ -357,4 +357,46 @@ const people = [
 // eslint-disable-next-line no-console
 console.log(people); // you can remove it
 
-// write your code here
+const dashboard = document.querySelector('.dashboard');
+
+people.forEach((element) => {
+  const newString = document.createElement('tr');
+  const nameCell = document.createElement('td');
+
+  nameCell.textContent = element.name;
+
+  const sexCell = document.createElement('td');
+
+  if (element.sex === 'm') {
+    sexCell.textContent = 'Male';
+  }
+
+  if (element.sex === 'f') {
+    sexCell.textContent = 'Female';
+  }
+
+  const dateOfBirth = document.createElement('td');
+
+  dateOfBirth.textContent = element.born;
+
+  const dateOfDied = document.createElement('td');
+
+  dateOfDied.textContent = element.died;
```

Comment:
Consider renaming 'dateOfDied' to 'dateOfDeath' for better readability and consistency with common terminology.